# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxUserFontSet.cpp
# Commit: ebecfbeb55c6
# Full Hash: ebecfbeb55c6fcbd730c2781cadfd850b26e4a26
# Author: Cameron McCormack <cam@mcc.id.au>
# Date: 2019-06-07 09:47:19
# Regressor Bug: 1490792
# File Overlap Count: 1
# Description:
#   Bug 1490792 - Part 2: Make LoadPlatformFont length argument not a reference. r=jfkthame
#   
#   We don't write to it.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D33895
# ==============================================================================

diff -r 478ac1179fa6 -r ebecfbeb55c6 gfx/thebes/gfxUserFontSet.cpp
--- a/gfx/thebes/gfxUserFontSet.cpp	Thu Jun 06 18:23:27 2019 +0000
+++ b/gfx/thebes/gfxUserFontSet.cpp	Thu Jun 06 18:23:29 2019 +0000
@@ -667,7 +667,7 @@
 MOZ_DEFINE_MALLOC_SIZE_OF_ON_ALLOC(UserFontMallocSizeOfOnAlloc)
 
 bool gfxUserFontEntry::LoadPlatformFont(const uint8_t* aFontData,
-                                        uint32_t& aLength) {
+                                        uint32_t aLength) {
   AUTO_PROFILER_LABEL("gfxUserFontEntry::LoadPlatformFont", OTHER);
   NS_ASSERTION((mUserFontLoadState == STATUS_NOT_LOADED ||
                 mUserFontLoadState == STATUS_LOAD_PENDING ||